tests/fixture.ts

```diff
@@ -0,0 +1,36 @@
+import * as avs_pb from "../grpc_codegen/avs_pb";
+import { UlidMonotonic } from "id128";
+import { requireEnvVar } from "./utils";
+
+// Dummy private key for determinictics test
+export const TEST_PRIVATE_KEY = "0x677240d3b2d8a167e536969785de1bb9f893ed0593682dc8c4dbaf8980d325ee"
```

Comment:
To make test written easiser and deterministic I used a hard code dummy private key. It helps:

easiser to write assertion for a smart wallet with salt. With dynamic private key, we cannot know ahead of time and have to make RPC request to get it, which reduce reliabily of the test.

deterministic because the output is pretty muched fixed and preditacable for our test.

Reply:
oh I remember why I chose to use an env variable: if we all use the same wallet address, runs in parallel will mess up the tests. For example, local dev running could colide with Github PR runs.

Reply:
Let’s separate the salt validation into a dedicated file and use this constant private key.

Reply:
updated.